components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/constants/CliConstants.java

```diff
@@ -75,20 +77,26 @@
     public static final String SYS_PROP_USER_DIR = "user.dir";
     public static final String SYS_PROP_CURRENT_DIR = "current.dir";
     public static final String SYS_PROP_SECURITY = "security";
+    public static final String WSO2 = "wso2";
+    public static final String MGW = "mgw";
     public static final String MICRO_GW = "micro-gw";
+    public static final String MICRO_GW_CONF_FILE = "micro-gw.conf";
     public static final String KEEP_FILE = ".keep";
     public static final String BALLERINA_TOML_FILE = "Ballerina.toml";
 
     public static final String LOGGING_PROPERTIES_FILENAME = "logging.properties";
     public static final Pattern SYS_PROP_PATTERN = Pattern.compile("\\$\\{([^}]*)}");
 
+    public static final String GET = "GET";
```

Comment:
We should be able to reuse `org.apache.http.client.methods.HttpGet.METHOD_NAME`

Reply:
fixed. thanks